api4/oauth.go

```diff
@@ -530,7 +530,10 @@ func completeOAuth(c *Context, w http.ResponseWriter, r *http.Request) {
 
 		c.App.Session = *session
 
-		redirectUrl = c.GetSiteURLHeader()
+		var ok bool
+		if redirectUrl, ok = props["redirect_to"]; !ok {
```

Comment:
While it's a bit less compact, could you change this around slightly to make it more obvious what's going on here? It took me a few minutes to realize that `redirectUrl` is actually modified in either case here.
```go
if _, ok = props["redirect_to"]; ok {
    redirectUrl = props["redirect_to"]
} else {
    redirectUrl = c.GetSiteURLHeader()
}
```

Reply:
Good point, and it's actually cleaner that way since I can switch to `_, ok := ...` instead of declaring `ok` before.